share/completions/firefox.fish

```diff
@@ -0,0 +1,41 @@
+# X11 options
+complete -c firefox -l display -r -d "X display to use"
+complete -c firefox -l sync -d "Make X calls synchronous"
+complete -c firefox -l g-fatal-warnings -d "Make all warnings fatal"
+
+# Firefox options
+complete -c firefox -s h -l help -d "Print this message"
+complete -c firefox -s v -l version -d "Print Firefox version"
+complete -c firefox -l full-version -d "Print Firefox version, build and platform build ids"
+complete -c firefox -s P -r -d "Start this profile" -fa '(string replace -rf "Name=(.*)" \'$1\' < ~/.mozilla/firefox/profiles.ini)'
+complete -c firefox -l profile -r -d "Start with profile at <path>"
+complete -c firefox -l migration -d "Start with migration wizard"
+complete -c firefox -l ProfileManager -d "Start with ProfileManager"
+complete -c firefox -l no-remote -d "Do not accept or send remote commands implies --new-instance"
+complete -c firefox -f -l new-instance -d "Open new instance, not a new window in running instance"
+complete -c firefox -f -l safe-mode -d "Disables extensions and themes for this session"
+complete -c firefox -f -l allow-downgrade -d "Allows downgrading a profile"
+complete -c firefox -l MOZ_LOG -r -d "Treated as MOZ_LOG=<modules> environment variable, overrides it"
```

Comment:
small misunderstanding: I meant only the --MOZ_LOG should have `-f`. Not the other flags like `--new-instance`, they don't take any parameters anyway

I'll correct both and merge. Thanks for the contribution!

Reply:
Thank you.